src/main/java/com/rcv/RawContestConfig.java

```diff
@@ -30,78 +30,43 @@
 @JsonIgnoreProperties(ignoreUnknown = true)
 public class RawContestConfig {
 
-  // directory for output files
-  public String outputDirectory;
-  // contest name
-  public String contestName;
-  // contest jurisdiction
-  public String contestJurisdiction;
-  // contest office
-  public String contestOffice;
-  // contest date
-  public String contestDate;
-  // should we report round-by-round results by precinct also?
-  public boolean tabulateByPrecinct;
-  // rules object
-  public ContestRules rules;
-  // list of all Candidates
-  public List<Candidate> candidates;
+  // contest settings object
+  public ContestSettings settings;
```

Comment:
The distinction between the words "config", "settings", and "rules" is not intuitive. Can we find a term that's more descriptive than "settings"? It seems like everything under "settings" is related to output. How about "outputSettings"?

Reply:
Done! Cowboy merging...